src/store/ChartStore.js

```diff
@@ -297,11 +302,20 @@ class ChartStore {
                 }
             } catch (e) { width = yax.width; } // Firefox doesn't like this in hidden iframe
 
-            let x = this.width - width;
+            width -= 14;
```

Comment:
@shayanbinary would you please comment what is `14` that is deducted from `width`?